tools/building.py

```diff
@@ -476,17 +476,6 @@ def check_closure_compiler(cmd, args, env, allowed_to_fail):
   return True
 
 
-# Remove this once we require python3.7 and can use std.isascii.
-# See: https://docs.python.org/3/library/stdtypes.html#str.isascii
```

Comment:
If you are proposing we drop support for python3.6 can you open a bug for that where we can discuss the costs/benefits?